src/System.Data.Odbc/dir.props

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.1.2.0</AssemblyVersion>
```

Comment:
Since this is the first time this assembly is being introduced in .Net Core, this version can be `4.0.0.0`